Simple-User-Management-API/Simple-User-Management-API/Models/UserManagementContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Simple_User_Management_API.Models
+{
+    public class UserManagementContext : DbContext
+    {
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer("Server=localhost;Database=UserManagementDB;Trusted_Connection=True;");
```

Comment:
Move to configuration, use IOptions pattern
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1